Library/Formula/v8.rb

```diff
@@ -76,8 +87,18 @@ def install
     (buildpath/"testing/gtest").install resource("gtest")
     (buildpath/"tools/clang").install resource("clang")
 
+    i18nsupport="i18nsupport=off"
+    if build.with? "icu4c"
+      i18nsupport="i18nsupport=on"
+      
+      # Patch needs icu-config on our path
+      icu4c = Formula["icu4c"]
+      ENV.append "PATH", icu4c.opt_bin
```

Comment:
This will happen automatically inside `brew`; We deliberately put `keg_only` dependencies into the `$PATH` when they're invoked.